co2eq/purchase/index.test.js

```diff
@@ -155,3 +167,44 @@ test(`test non-monetary units (in kg)`, () => {
   expect(modelCanRun(activity)).toBeTruthy();
   expect(carbonEmissions(activity)).toBeCloseTo(92.5);
 });
+
+test(`test equivalence of activityType=ACTIVITY_TYPE_PURCHASE and activityType=ACTIVITY_TYPE_MEAL`, () => {
+  const activity = {
+    activityType: ACTIVITY_TYPE_PURCHASE,
+    lineItems: [
+      { identifier: PURCHASE_CATEGORY_FOOD_SERVING_SERVICES, unit: UNIT_MONETARY_EUR, value: 10 },
+    ],
+    costCurrency: UNIT_MONETARY_EUR,
+    costAmount: 10,
+  };
+  expect(modelCanRun(activity)).toBeTruthy();
+  expect(carbonEmissions(activity)).toBeCloseTo(
+    carbonEmissions({ ...activity, activityType: ACTIVITY_TYPE_MEAL })
+  );
+});
+const TRANSPORTATION_MODE_TO_PURCHASE_IDENTIFIER = {
```

Comment:
@pierresegonne this mapping is the one we should probably extend or encode as part of https://github.com/tmrowco/bloom/issues/97

Reply:
Yes nice first step :)